packages/cache/src/index.ts

```diff
@@ -1 +1,19 @@
 export { buildCache } from './cache.js';
+export type {
+  Cache,
+  CacheTransaction,
+  LiveCacheTransaction,
+  CommittingTransaction,
+  CacheEntry,
+  CacheEntryState,
+  CacheKeyValue,
+  EntityMergeStrategy,
+  RevisionMergeStrategy,
+  CachedEntityRevision,
+  ExpirationPolicy,
+  CacheOptions,
+  DefaultRegistry,
+  CacheDebugAPIs,
+  LruCache,
+  CacheTransactionDebugAPIs,
+} from '../types/api.js';
```

Comment:
this is great thanks

Reply:
it would be great to have testing for the types here https://github.com/data-eden/data-eden/pull/91

Reply:
Will do, was planning to address this as part of this ticket https://github.com/data-eden/data-eden/issues/71